wellregistry/registry/tests/admin/test_monitoring_location.py

```diff
@@ -53,41 +53,27 @@ def setUp(self):
         self.admin = MonitoringLocationAdmin(MonitoringLocation, self.site)
 
     def test_site_id(self):
-        reg_entry = MonitoringLocation.objects.get(site_no='44445555',
-                                                   agency='ADWR')
+        reg_entry = MonitoringLocation.objects.get(site_no='44445555',agency='ADWR')
         site_id = MonitoringLocationAdmin.site_id(reg_entry)
 
         self.assertEqual(site_id, "ADWR:44445555")
 
-    def test_download_monitoring_locations(self):
```

Comment:
Seems like we still should have been able to use this test (perhaps with some test fixture modifications. This should be added back in or a similar test to test this action. This is particularly important since we are using a custom query.

Reply:
I ran many testes and modifications, and a whole different way of mocking up the data
because things like request and body don't have the required data from HttpRequest (i have tried adding params, but, each time more was needed)
and, the call to def test_download_monitoring_locations(self): calls
 response = self.admin.download_monitoring_locations(request, MonitoringLocation.objects.all())
  instead of the raw query

Reply:
i will work on a valid test.. Thanks again

Reply:
add the following tests
test_get_raw_queryset_with_superuser
test_get_raw_queryset_with_adwr_user
test_get_where_clause_with_superuser
test_get_where_clause_with_adwr_user
test_get_user_groups_with_superuser
test_user_groups_with_adwr_user